runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -95,7 +95,7 @@
 
 	//NPC messages
 	private static final Pattern NPC_ASSIGN_MESSAGE = Pattern.compile(".*Your new task is to kill\\s*(\\d*) (.*)\\.");
-	private static final Pattern NPC_ASSIGN_BOSS_MESSAGE = Pattern.compile("^Excellent. You're now assigned to kill (.*) (\\d+) times.*Your reward point tally is (.*)\\.$");
+	private static final Pattern NPC_ASSIGN_BOSS_MESSAGE = Pattern.compile("^Excellent. You're now assigned to kill (?:the )?(.*) (\\d+) times.*Your reward point tally is (.*)\\.$");
```

Comment:
Will this strip out Uppercase `the`?

Reply:
do they use uppercase the?

Reply:
Must be a disconnect between the infobox title then. The first letter is uppercase vs the lowercase of the chat message.

Reply:
Oh just noticed, yeah no clue why first letter is uppercase, maybe we do it somewhere in slay plugin